packages/react-theme/src/types.ts

```diff
@@ -77,6 +77,11 @@ export type NeutralColorTokens = {
   neutralShadowKeyDarker: string;
 };
 
+/**
+ * A ramp of size values.
+ */
+export type SizeValue = 'smallest' | 'smaller' | 'small' | 'medium' | 'large' | 'larger' | 'largest';
```

Comment:
IMO this should not be a part of `react-theme`, at least now. Can you please move this type to the package itself (i.e. `react-badge`) for now?

Reply:
Moved to `Badge` for now